webbeans-impl/src/main/java/org/apache/webbeans/config/BeansDeployer.java

```diff
@@ -1756,6 +1759,35 @@ protected void deployFromXML(ScannerService scanner)
         }
 
         logger.fine("Deploying configurations from XML has ended successfully.");
+
+        try
+        {
+            final URL url = new URL("openwebbeans", null, 0, "cdi-standalone", new URLStreamHandler()
+            {
+                @Override
+                protected URLConnection openConnection(URL u) throws IOException
+                {
+                    return null;
+                }
+            });
+
+            final BeanArchiveInformation beanArchiveInformation = beanArchiveService.getBeanArchiveInformation(url);
+            if (beanArchiveInformation != null)
+            {
+                configureDecorators(url, beanArchiveInformation.getDecorators());
+                configureInterceptors(url, beanArchiveInformation.getInterceptors());
+                configureAlternatives(url, beanArchiveInformation.getAlternativeClasses(), false);
+                configureAlternatives(url, beanArchiveInformation.getAlternativeStereotypes(), true);
+                configureAllowProxying(url, beanArchiveInformation.getAllowProxyingClasses());
+            }
+
+            logger.fine("Deploying embedded configurations has ended successfully.");
+        }
+        catch (Exception e)
+        {
+            logger.info("Error occurred: " + e.getMessage());
+            e.printStackTrace();
```

Comment:
I guess the e.printStackTrace is a left over?

Reply:
Hehe, yes, I forgot to remove that in my excitement to open the PR. :-)